### DISCUSSION TITLE
new clipmap based idea

### DISCUSSION THREAD

**Discussion by octanejohn** [TokisanGames/Terrain3D#347] [Questions] [answered]
what is the diffence and its worth it?
Uniform Distance-Dependent Level of Detail and Chunked Clipmaps https://dl.gi.de/bitstreams/cf03f446-82fe-424f-a289-118f624a0fae/download

**Comment by TokisanGames** [comment, marked as answer]
UDLOD was already brought up in #228 (and see #158). Follow up in those tickets.
